tests/test_server.py

```diff
@@ -225,13 +254,18 @@ async def test_get_trials(mcp: OptunaMCP) -> None:
     mcp.study.ask()
     result = await mcp.call_tool("get_trials", arguments={})
     assert isinstance(result, Sequence)
-    assert len(result) == 1
-    assert isinstance(result[0], TextContent)
-    lines = result[0].text.strip().split("\n")
-    assert len(lines) == 3
-    assert lines[0] == ("Trials: ")
-    assert lines[1].startswith(",number")
-    assert lines[2].startswith("0,0")
+    assert len(result) == 2
+    assert isinstance(result[0], list)
+    assert isinstance(result[0][0], TextContent)
+    assert isinstance(result[1], dict)
+    lines_from_text = result[0][0].text.strip().split("\n")
+    lines_from_dict = result[1]["result"].strip().split("\n")
+
+    for lines in (lines_from_text, lines_from_dict):
+        assert len(lines) == 3
+        assert lines[0] == ("Trials: ")
+        assert lines[1].startswith(",number")
+        assert lines[2].startswith("0,0")
```

Comment:
Enabling the `structured_output` at `mcp.tool` might make the output more complicated than the current main.